src/test/MockMultiplier.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import {IMultiplier} from "src/interfaces/IVotingMultipliers.sol";
+
+/// @title MockMultiplier
+/// @notice A mock contract implementing the IMultiplier interface for testing purposes
+contract MockMultiplier is IMultiplier {
+    uint256 private _multiplyingFactor;
+    uint256 private _validUntil;
+
+    /// @notice Sets the multiplying factor and valid until block for testing
+    /// @param factor The multiplying factor to set
+    /// @param validUntilBlock The block number until which the multiplier is valid
+    function setMultiplier(uint256 factor, uint256 validUntilBlock) external {
```

Comment:
underscore for internal vars / function scoped vars / parameters; `_factor, _validUntilBlock`

Reply:
addressed with 6e78550